source/_integrations/media_player.markdown

```diff
@@ -73,6 +73,13 @@ Currently only supported on [Sonos](/integrations/sonos), [Spotify](/integration
 | `entity_id`            |       no | Target a specific media player. For example `media_player.spotify`|
 | `shuffle`              |       no | `true`/`false` for enabling/disabling shuffle        |
 
+#### Service `media_player.repeat_set`
+
+| Service data attribute | Optional | Description                                          |
+| ---------------------- | -------- | ---------------------------------------------------- |
+| `entity_id`            |       no | Target a specific media player. For example `media_player.kitchen`|
```

Comment:
Entity ID is optional in this case. As this is created using `async_register_entity_service`, it could be targeting an area instead.

```suggestion
| `entity_id`            |       yes | Target a specific media player. For example `media_player.kitchen`|
```

Reply:
That was an unfortunate change. However, it is optional.

Reply:
I don't know what to do here; feel free to commit the suggestion if you prefer correctness over consistency.

Reply:
It is technical documentation, yes, we should prefer correctness.